data/exploits/CVE-2021-3156/exploit.c

```diff
@@ -36,32 +36,44 @@
 #define SUDOEDIT_PATH "/usr/bin/sudoedit"
 
 typedef struct {
-    char *target_name;
+    char *name;
     char *sudoedit_path;
     uint32_t smash_len_a;
     uint32_t smash_len_b;
     uint32_t null_stomp_len;
     uint32_t lc_all_len;
+    char *overwrite_path;
```

Comment:
overwrite_path seems unnecessary, you could consider just using the lib_path local variable directly, e.g:
`s_envp[envp_pos++] = lib_path`

Reply:
That's what I had done originally, but then I thought it made more sense to keep all of the target information together in the `target_t` struct. While the `sudoedit_path` is static, all the user-defined options make their way into there.